pkg/api/v1beta2/types.go

```diff
@@ -556,7 +556,7 @@ type Service struct {
 	// PortalIP is usually assigned by the master.  If specified by the user
 	// we will try to respect it or else fail the request.  This field can
 	// not be changed by updates.
-	PortalIP string `json:"portalIP,omitempty" description:"IP address of the service; usually assigned by the system; if specified, it will be allocated to the service if unused, and creation of the service will fail otherwise; cannot be updated"`
+	PortalIP string `json:"portalIP,omitempty" description:"IP address of the service; usually assigned by the system; if specified, it will be allocated to the service if unused, and creation of the service will fail otherwise; cannot be updated; cannot be updated"`
```

Comment:
PortalIP already have the "cannot be updated" sentence, that's why it didn't show up on diff. v1beta2 shows this because in the process of replacing immutable with "cannot be updated", I accidentally typed this twice (there are two "cannot be updated" above. This has been corrected. 

@thockin, I checked the update validation code of Service and didn't notice any field that cannot be updated. Please let me know if I have missed anything. Thanks!